ride_share.rb

```diff
@@ -0,0 +1,144 @@
+def print_number_rides(my_hash)
+  my_hash.each do |driver, rides|
+    total_rides = rides.length
+    print "#{driver} did #{total_rides} "
+
+    if total_rides == 1
+      puts "ride."
+    else
+      puts "rides."
+    end
+  end
+end
+
+def get_drivers_earnings(my_hash)
```

Comment:
I really appreciate how you used these helper methods! It's very clearly laid out!